### ISSUE TITLE
Use <button> or <a href="#" role=button">

### ISSUE THREAD

**Issue by Malvoz** [Igor-Vladyka/leaflet.browser.print#102]
The `<a>` print button is not keyboard focusable/operable (missing `href="#"`). Additionally, it is announced as a "link" using a screen reader (missing `role="button"`). Otherwise `<button type="button">` is recommended, but requires additional styling to reset browser defaults.

See:

- https://github.com/Leaflet/Leaflet/blob/bdd7ebf836201550f6d173943b728098b83f40d5/src/control/Control.Zoom.js#L85-L91
- https://www.w3.org/TR/2016/WD-wai-aria-practices-1.1-20160317/examples/button/button.html

**Comment by Malvoz** [comment]
Also see the <span title="Web Accessibility Initiative - Accessible Rich Internet Applications">WAI-ARIA</span> Authoring Practices [recommended keyboard interactions](https://www.w3.org/TR/wai-aria-practices/#keyboard-interaction-3) for [buttons](https://www.w3.org/TR/wai-aria-practices/#button).

**Comment by Malvoz** [comment]
Other potentially helpful resources:
- [Leaflet Guideline: Test your maps](https://leafletjs.com/SlavaUkraini/examples/accessibility/#test-your-maps)
- [Leaflet Plugin Authoring Guide: Content Accessibility](https://github.com/Leaflet/Leaflet/blob/main/PLUGIN-GUIDE.md#content-accessibility)
